crossbeam-queue/src/lib.rs

```diff
@@ -29,6 +30,7 @@ cfg_if! {
 
 extern crate crossbeam_utils;
 
+#[cfg_attr(feature = "nightly", cfg(target_has_atomic = "ptr"))]
```

Comment:
In stable+std+thumbv6m-none-eabi, what happens?  I guess the compilation fails, and wonder if it's the desired behavior.

Reply:
I don't think that the standard library is available for `thumbv6m-none-eabi`, so the compilation would fail either way. As far as I know, it is also not possible to create a `no_std` binary that uses the `alloc` library on stable yet, because the required `#[alloc_error_handler]` is not stable yet.